internal/backends/winit/lib.rs

```diff
@@ -384,17 +384,14 @@ impl BackendBuilder {
             }
             #[cfg(feature = "renderer-femtovg-wgpu")]
             (Some("femtovg-wgpu"), maybe_graphics_api) => {
```

Comment:
Style suggestion, but not sure if worth it:

```suggestion
            (Some("femtovg-wgpu"), Some(_api)) => {
```

And then another branch for the None case would give one level less.
But as i said, this might actually not be simpler so feel free to ignore.